tools/appliance/systemvmtemplate/scripts/configure_conntrack.sh

```diff
@@ -34,8 +34,6 @@ function load_conntrack_modules() {
   grep nf_conntrack_ipv4 /etc/modules && return
 
   cat >> /etc/modules << EOF
-nf_conntrack_ipv4
-nf_conntrack_ipv6
```

Comment:
@Pearl1594 any reason to remove these?

Reply:
there is only an nf_conntrack module, it is because of these unknown modules that we see an error during systemvm bootup
![image](https://user-images.githubusercontent.com/10495417/130806187-ac5896b4-acb6-415d-986a-f356643a6fef.png)